cmd/gateway_server/gateway.pb.go

```diff
@@ -0,0 +1,285 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: gateway.proto
+
+package gateway
+
+import (
+	context "context"
+	fmt "fmt"
+	proto "github.com/golang/protobuf/proto"
+	grpc "google.golang.org/grpc"
+	math "math"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
FYI my local attempt at building this branch failed on this line. Looks like we've got an old copy of protobuf vendored. Please:

1. Add `ignored = ["google.golang.org/grpc"]` to Gopkg.toml and include that change in this PR. (Because it's big and I don't want a big non-essential dep in here, per [my reasoning here](https://github.com/ampproject/amppackager/blob/master/CONTRIBUTING.md#new-dependencies).)
2. Run `dep ensure -update github.com/golang/protobuf` and include the changed/added files in this PR.